core/bind/core_bind.cpp

```diff
@@ -1177,18 +1177,10 @@ _OS *_OS::singleton = NULL;
 
 void _OS::_bind_methods() {
 
-	//ClassDB::bind_method(D_METHOD("get_mouse_position"),&_OS::get_mouse_position);
-	//ClassDB::bind_method(D_METHOD("is_mouse_grab_enabled"),&_OS::is_mouse_grab_enabled);
-
 	ClassDB::bind_method(D_METHOD("set_clipboard", "clipboard"), &_OS::set_clipboard);
 	ClassDB::bind_method(D_METHOD("get_clipboard"), &_OS::get_clipboard);
 
 	//will not delete for now, just unexpose
```

Comment:
This should be deleted too, because it refers to the commented code below.

Reply:
This was marked as resolved but I confirm the comment should be removed.
The matching methods should also be removed from the OS class (I think this commit does it already).